app/assets/stylesheets/application.scss

```diff
@@ -2,13 +2,92 @@
 @import "bootstrap-sprockets";
 @import "bootstrap";
 
+body {
+  background: #F9F9F9
+}
+
+#face-img {
+  position: relative;
+}
+
+#face-img img{
+  height: 400px;
+}
+
+#logo {
+  display: inline;
+  margin-top: 1em;
+  justify-content: center;
+  font-family: monospace;
+}
+
 div#intro {
-    margin-top: 100px;
-    width: 80%;
-    margin-left: 20%;
+  margin-top: 100px;
+  width: 80%;
+  margin-left: 20%;
 }
 
 div#foot-contact {
-    position: fixed;
-    bottom: 0px;
+  position: fixed;
+  bottom: 0px;
+}
+
+div#face-header {
+  display: inline-flex;
+  width: 100%;
+  justify: right;
+}
+
+
+.icons {
+  width: 500px;
+  margin: 0px;
+  float: right;
+}
+
+.icons a {
+  display: inline-block;
+  width: 24px;
+  height: 24px;
+  margin: 10px;
+  vertical-align: middle;
+  -o-transition: all .3s;
+  -moz-transition: all .3s;
+  -webkit-transition: all .3s;
+  -ms-transition: all .3s;
+}
+
+.icons a.twitter {
+  background: url(images/twitter.png)
+  left top no-repeat;
+}
+
+.icons a.vimeo {
+  background: url(./vimeo.png)
+  left top no-repeat;
+}
+
+.icons a.linkedin {
+  background: url(./linkedin.png)
+  left top no-repeat;
+}
+
+.icons a.facebook {
+background: url(./facebook.png)
```

Comment:
Line should be indented 2 spaces, but was indented 0 spaces<br>URLs should be enclosed in quotes